NRLMMD-GEOIPS/geoips
# 29 convert python script docstrings to numpy docstring format

# Related Issues
fixes NRLMMD-GEOIPS/geoips#[XXX]

# Testing Instructions
Check for valid numpy docstrings:
```
find ./geoips/image_utils/ -type f -name "*.py" -exec pydocstyle --convention=numpy  "{}" \;
```

Check that basic test script still runs:
```
geoips/tests/test_base_install_low_memory.sh
```

# Summary
NRLMMD-GEOIPS/geoips#29: 2022-08-28, Update image_utils to numpy docstrings

### Documentation
* Update image_utils directory for proper numpy style docstrings
    * **geoips/image_utils/__init__.py**
    * **geoips/image_utils/colormap_utils.py**
    * **geoips/image_utils/maps.py**
    * **geoips/image_utils/mpl_utils.py**
* **geoips/compare_outputs.py** Update imagemagick compare metric from rmse to ae + fuzz 1%

# Output
No output from pydocstyle:
```
(geoips_conda) find ./geoips/image_utils/ -type f -name "*.py" -exec pydocstyle --convention=numpy  "{}" \;
(geoips_conda)
```

0 return from test script:
```
Package: geoips_base
Total run time: 145 seconds
Number data types run: 3
Number data types failed: 0
```
